core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -768,7 +769,18 @@ default HttpResponse mapHeaders(Function<? super ResponseHeaders, ? extends Resp
                 }
             }
             return obj;
-        });
+        })
```

Comment:
is this the desired indentation, or should the line be `}).mapError() (obj -> {`?

Reply:
removed